src/utils/stringHelper.test.ts

```diff
@@ -12,4 +13,19 @@ describe('stringHelper', () => {
     expect(capitalizeName('léonardo di caprio')).toEqual('Léonardo Di Caprio');
     expect(capitalizeName('" \'')).toEqual('" \'');
   });
+
+  it('can get the inner child of an element as a string', () => {
+    expect(getLabelFromChildren(getParsedLanguageFromText('<h1>This is my message</h1>'))).toEqual(
+      'This is my message',
+    );
+    expect(getLabelFromChildren(getParsedLanguageFromText('<span>This is my message</span>'))).toEqual(
+      'This is my message',
+    );
+    expect(getLabelFromChildren(getParsedLanguageFromText('<div><span>This is my message</span></div>'))).toEqual(
+      'This is my message',
+    );
+
+    const myReactNode: React.ReactNode = getParsedLanguageFromText('<div><span>This is my message</span></div>');
+    expect(getLabelFromChildren(myReactNode)).toEqual('This is my message');
+  });
```

Comment:
Cool stuff! Does it work for `<div><span>text1</span><span>text2</span></div>` as well? Might add a test-case for that to be sure, and document what is supposed to happen in that case.

Reply:
It does unfortunately not work for that example. The output is an array with `[<span>text1</span>, <span>text2</span>]`. Mabye the function could be updated to run again for each item in the array? 🤔

Reply:
An array? Huh, that was unexpected - the TypeScript return type says only strings will be returned. I would expect an empty object (or null) as input would also return a string, then (empty string, maybe). I would suggest adding such cases to the test and adjust the typescript return type (or implement it such that the example input returns `text1text2` as expected).